src/scss/header.scss

```
@@ -69,6 +69,30 @@ header {
   line-height: 1.2;
 }
 
+a.skip-main {
+  left:-999px;
+  position:absolute;
+  top:auto;
+  width:1px;
+  height:1px;
+  overflow:hidden;
+  z-index:-999;
+}
+a.skip-main:focus, a.skip-main:active {
+  color: #000;
+  background-color:#fff;
+  left: -30%;
+  top: 60px;
+  width: 150px;
```

This width may want to be bigger if you accept the change above!
```suggestion
  width: 175px;
```